src/cli/clear.js

```
@@ -0,0 +1,92 @@
+// @flow
+// implementation of `sourcecred clear`
+
+import path from "path";
+import fs from "fs";
+
+import dedent from "../util/dedent";
+import {type Command} from "./command";
+import * as Common from "./common";
+
+function usage(print: (string) => void): void {
+  print(
+    dedent`\
+    usage: sourcecred clear --all
+           sourcecred clear --cache
+           sourcecred clear --help
+
+    Remove the SOURCECRED_DIRECTORY, i.e. the directory where data, caches,
+    registries, etc. owned by SourceCred are stored.
+
+    Arguments:
+        --all
+            remove entire SOURCECRED_DIRECTORY
+
+        --cache
+            remove only the SourcCred cache directory
+
+        --help
+            Show this help message and exit, as 'sourcecred help export-graph'.
+
+    Environment Variables:
+        SOURCECRED_DIRECTORY
+            Directory owned by SourceCred, in which data, caches,
+            registries, etc. are stored. Optional: defaults to a
+            directory 'sourcecred' under your OS's temporary directory;
+            namely:
+                ${Common.defaultSourcecredDirectory()}
+
+    `.trimRight()
+  );
+}
+
+function die(std, message) {
+  std.err("fatal: " + message);
+  std.err("fatal: run 'sourcecred help clear' for help");
+  return 1;
+}
+
+export const clear: Command = async (args, std) => {
+  switch (args[0]) {
```

Should fail if there is not exactly one argument, rather than silently
swallowing all but the first.

---

I've updated that function